src/lib/custom-animations/GestureActionView.tsx

```diff
@@ -0,0 +1,404 @@
+import React from 'react'
+import {ColorValue, Dimensions, StyleSheet, View} from 'react-native'
+import {Gesture, GestureDetector} from 'react-native-gesture-handler'
+import Animated, {
+  clamp,
+  interpolate,
+  interpolateColor,
+  runOnJS,
+  useAnimatedReaction,
+  useAnimatedStyle,
+  useDerivedValue,
+  useSharedValue,
+  withSequence,
+  withTiming,
+} from 'react-native-reanimated'
+
+import {useHaptics} from '#/lib/haptics'
+import absoluteFill = StyleSheet.absoluteFill
```

Comment:
not this again haha

Reply:
Oh shit, thanks for pointing that out lol.